compute_tools/src/sql/finalize_drop_subscriptions.sql

```diff
@@ -0,0 +1,21 @@
+DO $$
+BEGIN
+    IF NOT EXISTS(
+        SELECT 1
+        FROM pg_catalog.pg_tables
+        WHERE tablename = 'drop_subscriptions_done'
+        AND schemaname = 'neon_migration'
+    )
+    THEN
+        CREATE SCHEMA IF NOT EXISTS neon_migration;
+        ALTER SCHEMA neon_migration OWNER TO cloud_admin;
+        REVOKE ALL ON SCHEMA neon_migration FROM PUBLIC;
+
+        CREATE TABLE neon_migration.drop_subscriptions_done
+        (timeline_id text);
+    END IF;
+
+    INSERT INTO neon_migration.drop_subscriptions_done
+    VALUES (current_setting('neon.timeline_id'));
```

Comment:
Just inserting we will bloat this table if user creates branch of a branch OR users restore, because each time we will only append a new timeline here.

I think we can safely clean the table each time, but in a transaction block (which is probably ensured by procedure block, but not sure), like
```sql
BEGIN;
DELETE FROM neon_migration.drop_subscriptions_done
INSERT INTO neon_migration.drop_subscriptions_done
    VALUES (current_setting('neon.timeline_id'));
END;
```

That way, we only have the latest timeline in this table, which is what we actually need. And its size will be likely limited to a single page